### ISSUE TITLE
libmath-mpfr-perl FTBFS on ppc*: subnormal_doubles.t fails

### ISSUE THREAD

**Issue by gregoa** [sisyphus/math-mpfr#2]
We have the following bug reported to the Debian package of Math-MPFR
(https://bugs.debian.org/918673):

It doesn't seem to be a bug in the packaging, so you may want to take
a look. Thanks!

------8<-----------8<-----------8<-----------8<-----------8<-----

```
Source: libmath-mpfr-perl
Version: 4.05-1
Severity: serious
Tags: ftbfs

https://buildd.debian.org/status/package.php?p=libmath-mpfr-perl

...
Invalid type 'D' in pack at t/subnormal_doubles.t line 63.
t/subnormal_doubles.t ............ 
1..1
Dubious, test returned 255 (wstat 65280, 0xff00)
Failed 1/1 subtests 

Calling all cache freeing functions
Skipping tests 4, 5 and 6 - not relevant to this build
...
Test Summary Report
-------------------
t/subnormal_doubles.t          (Wstat: 65280 Tests: 0 Failed: 0)
  Non-zero exit status: 255
  Parse errors: Bad plan.  You planned 1 tests but ran 0.
Files=69, Tests=1137, 12 wallclock secs ( 0.29 usr  0.08 sys +  2.49 cusr  0.28 csys =  3.14 CPU)
Result: FAIL
Failed 1/69 test programs. 0/1137 subtests failed.
make[1]: *** [Makefile:1021: test_dynamic] Error 255


```
------8<-----------8<-----------8<-----------8<-----------8<-----

Thanks for considering,
  gregor herrmann,
  Debian Perl Group

**Comment by sisyphus** [comment]
Thanks gregor.
The failing test was intended only for -Duselongdouble powerpc builds of
perl.

The test script (t/subnormal_doubles.t) is now fixed in version 4.06, which
has been uploaded to cpan.

Cheers,
Rob

**Comment by gregoa** [comment]
On Wed, 09 Jan 2019 20:18:42 -0800, sisyphus wrote:

> The failing test was intended only for -Duselongdouble powerpc builds of
> perl.
> The test script (t/subnormal_doubles.t) is now fixed in version 4.06, which
> has been uploaded to cpan.

Thanks, much appreciated.
I've uploaded 4.06 to Debian, we can watch the build results coming
in at https://buildd.debian.org/status/package.php?p=libmath%2dmpfr%2dperl

**Comment by sisyphus** [comment]
Issue was fixed ... some time ago.
